### ISSUE TITLE
feat: healthcheck while building

### ISSUE THREAD

**Issue by dargmuesli** [nuxt/nuxt.js#14697]

**Comment by danielroe** [comment]
would you mind opening a nitro issue if you want? 🙏

nuxtathon closed @Norbiros

**Comment by dargmuesli** [comment]
Not at all! I thought a bit more about it and came up with this proposal:
- https://github.com/nitrojs/nitro/issues/4451
